Rubberduck.Parsing/Inspections/Resources/InspectionsUI.resx

```diff
@@ -790,6 +790,19 @@ If the parameter can be null, ignore this inspection result; passing a null valu
   <data name="IntegerDataTypeQuickFix" xml:space="preserve">
     <value>Change declaration type to Long</value>
   </data>
+  <data name="RemoveStopKeywordQuickFix" xml:space="preserve">
+    <value>Remove usage of the 'stop' keyword</value>
+  </data>
+  <data name="StopKeywordInspectionMeta" xml:space="preserve">
+    <value>The 'stop' keyword halts execution and brings up the debugger. Avoid its usage.</value>
+  </data>
+  <data name="StopKeywordInspectionName" xml:space="preserve">
+    <value>Stop keyword</value>
+  </data>
+  <data name="StopKeywordInspectionResultFormat" xml:space="preserve">
+    <value>Stop keyword detected</value>
+    <comment>{0} Property name</comment>
+  </data>
```

Comment:
I wonder how these i18n Strings got into the PR

Reply:
I don't know. You want me to do something with it?